scripts/proteinInteractionEBI/parseEBI.py

```diff
@@ -0,0 +1,333 @@
+        IDList = defLong[idStart+1:-1].split(", ")
+        termDic['references']= IDList     
+    
+    schemaPieceList = []
+    keyList = ["id", "def","references","parentClassName"]
+    
+    idString = termDic['id'][0]
+    dcid = id2className[idString]
+    
+    curLine = "Node: dcid:" + dcid
+    schemaPieceList.append(curLine)
+    
+    if idString in interactionTypeIdSet:
+        curLine = "typeOf: dcs:InteractionTypeEnum"
+    elif idString in detectionMethodIdSet:
+        curLine = "typeOf: dcs:InteractionDetectionMethodEnum"
+    elif idString in interactionSourceIdset:
+        curLine = "typeOf: dcs:InteractionSourceEnum"
+    else:
+        return None
+        
+    termDic["parentClassName"] = [id2className[node.value] for node in id2node[idString].parentList] 
+    schemaPieceList.append(curLine)
+    
+    curLine = "name: \"" + dcid + "\""
+    schemaPieceList.append(curLine)
+
+    '''
+    termDic:
+    id:  ['MI:0001']
+    name:  ['interaction detection method']
+    def:  ['Method to determine the interaction']
+    subset:  ['Drugable', 'PSI-MI_slim']
+    synonym:  ['"interaction detect" EXACT PSI-MI-short []']
+    relationship:  ['part_of MI:0000 ! molecular interaction']
+    references:  ['PMID:14755292']
+    parentClassName:  ['MolecularInteraction']
+    '''
+    
+    for key in keyList:
+            
+        if key=="def" and len(termDic[key])>0 :
+            curLine = "description: \"" + termDic[key][0][0].upper() + termDic[key][0][1:] +"\""
+            schemaPieceList.append(curLine)
+
+        elif key=="references" and len(termDic[key])>0:
+            itemList = []
+            for i in range(len(termDic[key])):
+                if termDic[key][i]!="": 
+                    curLine, newReferenceMap = get_references(termDic[key][i])
+                    if curLine:
+                        schemaPieceList.append(curLine)
+                    if newReferenceMap:
+                        newSourceMap[key] = newSourceMap[key].update(newReferenceMap)
+            
+        elif key=="id" and len(termDic[key])>0:       
+            curLine = "psimiID: \"" + termDic[key][0] + "\""
+            schemaPieceList.append(curLine)
+            
+        elif key=="parentClassName" and len(termDic[key])>0:
+            itemList = []
+            for i in range(len(termDic[key])):
+                itemList.append( "dcs:" + termDic[key][i])
+            curLine = "specializationOf: " +  ",".join(itemList)
+            schemaPieceList.append(curLine)
+            
+    curLine = "descriptionUrl: \"http://psidev.info/groups/controlled-vocabularies\""
+    schemaPieceList.append(curLine)
+    
+    return "\n".join(schemaPieceList), termDic['id'][0], dcid, newSourceMap
+
+
+def main(argv):
+    dbFile = argv[0]
+    with open(dbFile, 'r') as fp:
+        file = fp.read()
+    # clip exists in dcs already. Substitute with ClipInteraction
+    file = file.replace("name: clip\ndef", "name: clip interaction\ndef")
```

Comment:
Its good practice to be consistent with quotes throughout the file. Generally, I think prefer single to double.